pkg/cloudprovider/aws/suite_test.go

```diff
@@ -234,9 +229,8 @@ var _ = Describe("Allocation", func() {
 			Expect(fakeEC2API.CalledWithCreateFleetInput[0].LaunchTemplateConfigs[0].Overrides).To(
 				ContainElements(
 					&ec2.FleetLaunchTemplateOverridesRequest{
-						AvailabilityZone: aws.String("test-zone-1c"),
-						InstanceType:     aws.String("m5.large"),
-						SubnetId:         aws.String("test-subnet-3"),
+						InstanceType: aws.String("m5.large"),
```

Comment:
Since we're removing zones from these tests, can we also remove zones from line 209 where we pass them as DescribeSubnetOutputs?

Reply:
We actually still need those zones to see determine what zones the cluster spans.